Documentation/data-collection.md

```diff
@@ -245,6 +245,9 @@ data:
     # (@openshift/storage, OpenShift Storage team) Persistent Volume usage metrics: this is the number of volumes per plugin
     # and per volume type (filesystem/block)
     - '{__name__="cluster:kube_persistentvolume_plugin_type_counts:sum"}'
+    # (Advanced Cluster Management) visual_web_terminal_sessions_total is the count of Visual Web Terminal sessions created 
```

Comment:
Do you not have a team under openshift org we can ping in case we need to change/remove anything?

Reply:
I will have to check.  I think all of our Github teams (if that is what you mean) are under our `open-cluster-management` org.

Reply:
Yes we prefer a GitHub team if possible.

Reply:
OK, let me get the right one and I will update comment in the yaml with that.